src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -0,0 +1,40 @@
+package mate.controller.car;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.car.CarService;
+import mate.service.driver.DriverService;
+
+@WebServlet("/cars/addDriver")
```

Comment:
```suggestion
@WebServlet("/cars/drivers/add")
```
Let's compose your urls this way. Please follow common mistakes and fix it in all places